src/app/pages/OnDemandRadioPage/index.test.jsx

```diff
@@ -126,14 +103,83 @@ describe('OnDemand Radio Page ', () => {
     const { pageData: pageDataWithWithoutVideo } = await getInitialData(
       'some-ondemand-radio-path',
     );
-    const { getByText } = render(
-      createAssetPage(pageDataWithWithoutVideo, 'indonesia'),
-    );
+    const { getByText } = await renderPage({
+      pageData: pageDataWithWithoutVideo,
+      service: 'indonesia',
+    });
 
     expect(
       getByText(
         'Berita terbaru dari seluruh dunia dan ulasan peristiwa dari Indonesia. Juga berita olahraga terbaru dan berbeda setiap harinya.',
       ),
     ).toBeInTheDocument();
   });
+
+  it('should show the audio player on canonical', async () => {
+    const clonedKoreanPageData = clone(koreanPageData);
+    clonedKoreanPageData.content.blocks[0].versions[0].availableUntil = 9999999999999999999999999;
```

Comment:
```suggestion
    clonedKoreanPageData.content.blocks[0].versions[0].availableUntil = Infinity;
```

Reply:
@ryanmccombe when I change the data to that, it gets stored as null then the tests fail 😕

Reply:
Hmm okay, weird

Reply:
ah, because of this - `JSON.stringify(Infinity) // "null"`

Reply:
Ah makes sense.  The 9s are good - gonna annoy some developers when this unit test starts failing in the year 50,000

Reply:
Don't worry. I've put in my calendar to fix the unit test on Sat Nov 20 2286 17:46:39 GMT+0000 (Greenwich Mean Time)